client/src/app/components/stardust/Activity.tsx

```diff
@@ -0,0 +1,43 @@
+import { FunctionComponent } from 'react';
+import "./Transaction.scss";
+import { ActivityProps } from "./ActivityProps";
+
+export const Activity: FunctionComponent<ActivityProps> = ({ hash, date, action, status, price, tableFormat  }) => 
```

Comment:
Let's use `React.FC<ActivityProps>` so we have less imports.

Reply:
Done